src/templates/question_toolbar.html

```diff
@@ -9,8 +9,5 @@
     </div>
 </div>
 <% if (comment) { %>
-<div class="alert alert-info">
-  <button type="button" class="close" data-dismiss="alert">&times;</button>
-  <%= comment %>
-</div>
+<div class="alert alert-info fd-question-comment"><button type="button" class="close" data-dismiss="alert">&times;</button><%= comment %></div>
```

Comment:
is the formatting change a vellum style guide i'm not familiar with? just curious since the older formatting looks more readable to me.